client/less/map.less

```diff
@@ -91,16 +91,15 @@
 
 .map-buttons {
   margin-top: -10px;
-}
-.map-buttons button,
-.map-buttons .input-group{
-  width:300px;
-}
-
-.map-buttons .input-group{
-  margin-top: 15px;
-  margin-left: auto;
-  margin-right: auto;
+  & button,
+  & .input-group{
```

Comment:
it's a preprocessor function to summon the parent... 

You can also bundle pseudo-selectors with your mixins using this method. Here's the classic clearfix hack, rewritten as a mixin (& represents the current selector parent):

```CSS
.clearfix {
  display: block;
  zoom: 1;

  &:after {
    content: " ";
    display: block;
    font-size: 0;
    height: 0;
    clear: both;
    visibility: hidden;
  }
}
```